cmd/porter/bundle.go

```diff
@@ -77,6 +79,35 @@ func buildBuildCommand(p *porter.Porter) *cobra.Command {
 	return cmd
 }
 
+func buildBundleListCommand(p *porter.Porter) *cobra.Command {
+	opts := struct {
```

Comment:
Heads up that I made a mistake when I first did some of the original commands, defining the structs anonymously here, instead of as a type in pkg/porter like we do for other commands like Install.

It would be really nice at some point to fix these older commands to use the new and improved pattern, instead of adding more commands that use my old mistake. I'm fine with merging this, but if you feel motivated, as a follow up maybe we could fix this one and the other one you used as a pattern for this?

Reply:
Ah, ok.  Definitely, I'll start a follow-up.